hub/core/transform/test_transform.py

```diff
@@ -117,13 +120,32 @@ def add_link(sample_in, samples_out):
     samples_out.abc.append(hub.link(sample_in))
 
 
+@hub.compute
+def add_images(i, sample_out):
+    for i in range(5):
+        image = hub.read(get_dummy_data_path("images/flower.png"))
+        sample_out.append({"image": image})
+
+
 def check_target_array(ds, index, target):
     np.testing.assert_array_equal(
         ds.img[index].numpy(), target * np.ones((200, 200, 3))
     )
     np.testing.assert_array_equal(ds.label[index].numpy(), target * np.ones((1,)))
 
 
+def retrieve_objects_from_memory(object_type=hub.core.sample.Sample, save=False):
+    total_n_of_occurences = 0
+    list_to_return = []
+    gc_objects = gc.get_objects()
+    for item in gc_objects:
+        if isinstance(item, object_type):
+            if save:
```

Comment:
perhaps we can remove save as we're never passing it as true?

Reply:
yes, we can! I will remove it!